lib/src/participant/local.dart

```diff
@@ -271,6 +306,17 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
     await room.engine.sendDataPacket(packet);
   }
 
+  @override
+  EncryptionType get encryptionType {
```

Comment:
a bit of comments for this method would be helpful. or maybe rename to `firstTrackEncryptionType` to be clear?